src/ampAphroditeInterfaceFactory.js

```diff
@@ -1,9 +1,3 @@
-import {
```

Comment:
this change makes it semver-major; if we keep these and use the passed args as overrides when present, then it'd be semver-minor

Reply:
Ok, I'll do that